msal/src/main/java/com/microsoft/identity/client/PublicClientApplicationConfiguration.java

```diff
@@ -597,6 +602,14 @@ public void checkIntentFilterAddedToAppManifestForBrokerFlow() throws MsalClient
             return;
```

Comment:
I'm wondering if the change from this line until the end of the function belongs here.

It might make more sense for this to go into validateConfiguration(), or, specfically, validateRedirectUrl().

With that, we wouldn't need to put this special check in multiple places.